lib/test_broker.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:dsbroker/broker.dart';
+import 'package:dslink/dslink.dart';
+import 'package:dslink_dart_test/test_broker_node_provider.dart';
+import 'dart:convert';
+import 'dart:async';
+import 'dart:io';
+import 'src/config.dart';
+
+class TestBroker {
+  int httpPort;
+  int httpsPort;
+  TestBrokerNodeProvider broker;
+  DsHttpServer server;
+  BrokerDiscoveryClient discovery;
+  SecurityContext context = SecurityContext.defaultContext;
+
+  TestBroker(this.httpPort, this.httpsPort);
+
+  Future<Null> start() async {
+    var https = false;
+    var config = JSON.decode(testBrokerConfig);
```

Comment:
`JSON.decode(testBrokerConfig) as Map<String, String>` if that's the correct type. That'll enable the analyzer's help for the rest if the scope.